src/Workspaces/VisualBasic/Portable/Simplification/AbstractVisualBasicSimplifier.AbstractExpressionRewriter.vb

```diff
@@ -122,7 +139,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Simplification
                 Return newToken
             End Function
 
-            Protected Function SimplifyExpression(Of TExpression As ExpressionSyntax)(
+            Protected Function SimplifyExpression(Of TExpression As SyntaxNode)(
```

Comment:
Then we really need to change the name.  SimplifyExpression means to me that it will be getting an expression.  I have no problem with you calling this SimplifyExpressionOrSimpleArgumentOrFieldInitializer.  Or just having three helpers that each handle a single type.